azure/azure-how-to/instances/find-ubuntu-images.rst

```diff
@@ -112,6 +112,9 @@ Ubuntu Pro 24.04 LTS - Noble Numbat
    * - AMD64
      - Gen1
      - ``Canonical:ubuntu-24_04-lts:ubuntu-pro-gen1:latest``
+   * - AMD64
+     - Gen2
+     - ``Canonical:ubuntu-24_04-lts:ubuntu-pro-cvm:latest``
```

Comment:
I am not sure if we need to add this here (in the Pro offers section), as we have a separate section for CVM offers. Maybe it makes sense to add it to both.

Reply:
This is the crux of the issue that I have raised with alliances in how we have canonically "described" our Pro offerings. We are now rapidly generating Pro flavors of a range of base products. It used to simply just be only "Ubuntu Pro" - implicit as Pro Server - and a small set of extraneous Pro-only products (FIPS and CIS). But now we are producing (and promoting in many cases) the Pro variant over the base product.